internal/host/hostcommands/install_cmd.go

```diff
@@ -280,8 +280,11 @@ controls DHCP depending on the IP stack being used.
 func constructHostInstallerArgs(cluster *common.Cluster, host *models.Host, inventory *models.Inventory, infraEnv *common.InfraEnv, log logrus.FieldLogger) (string, error) {
 	var installerArgs []string
 	var err error
+	var hasStaticNetwork bool
 
-	hasStaticNetwork := (infraEnv != nil && infraEnv.StaticNetworkConfig != "") || cluster.StaticNetworkConfigured
+	if inventory == nil {
```

Comment:
added this check because we have a unit test for it: https://github.com/openshift/assisted-service/blob/master/internal/host/hostcommands/install_cmd_test.go#L1275-L1281
Before it was failing in `appendDHCPArgs` function, but if we have the inventory set to `nil` along with static  networking, we would get a panic.